base/linalg/cholesky.jl

```diff
@@ -491,6 +491,21 @@ function A_ldiv_B!(C::CholeskyPivoted, B::StridedMatrix)
     end
 end
 
+\(::Cholesky, ::RowVector) = throw(DimensionMismatch("Cannot left-divide matrix by transposed vector"))
+\(::CholeskyPivoted, ::RowVector) = throw(DimensionMismatch("Cannot left-divide matrix by transposed vector"))
+\(::Cholesky{<:Number}, ::RowVector{<:Number}) = throw(DimensionMismatch("Cannot left-divide matrix by transposed vector"))
```

Comment:
Now I understand the original question (lines 3 and 4 here alarmed me much less than lines 1 and 2 did) - @kshyatt the reason for that in bidiagonal was purely to avoid ambiguity errors, so only define such methods where necessary (but that seems irrelevant if we are deleting all 4 methods).